doc/api/util.md

```diff
@@ -46,7 +46,7 @@ environment variable. For example: `NODE_DEBUG=fs,net,tls`.
 
 ## util.deprecate(function, string)
 
-The `util.deprecate()` method wraps the given `function` in such a way that
+The `util.deprecate()` method wraps the given `function` or `class` in such a way that
```

Comment:
I think the backticks around "function" here, refer to the named parameter. If that's the case, I'm not sure it makes sense to put "class" in backticks too.